.github/workflows/process_opened_issues.yml

```diff
@@ -0,0 +1,17 @@
+name: Process (re)opened issues
+
+on:
+  issues:
+    types:
+      - reopened
+      - opened
+
+jobs:
+  add-to-project:
+    name: Adding issue to the IREE GitHub project
+    runs-on: ubuntu-latest
```

Comment:
this version should also be pinned, e.g. to ubuntu-20.04

https://opensource.google/documentation/reference/github/services#actions

> Docker images should always be run at a fixed version rather than "latest".